pkg/util/testing/wrappers.go

```diff
@@ -831,6 +831,15 @@ func (rf *ResourceFlavorWrapper) Obj() *kueue.ResourceFlavor {
 	return &rf.ResourceFlavor
 }
 
+// ObjectMetaLabel sets the label on the ResourceFlavor.
+func (rf *ResourceFlavorWrapper) ObjectMetaLabel(k, v string) *ResourceFlavorWrapper {
+	if rf.ObjectMeta.Labels == nil {
+		rf.ObjectMeta.Labels = map[string]string{}
+	}
+	rf.ObjectMeta.Labels[k] = v
+	return rf
+}
+
 // Label add a label kueue and value pair to the ResourceFlavor.
 func (rf *ResourceFlavorWrapper) Label(k, v string) *ResourceFlavorWrapper {
 	rf.Spec.NodeLabels[k] = v
```

Comment:
Rename this to `NodeLabel` or similar.

Reply:
it will take a lot of changes on project. Is it ok?

Reply:
I just thought maybe create another cleanup PR to fix it...

Reply:
Done